app/src/androidTest/java/com/group4sweng/scranplan/PlannerFragmentTest.java

```diff
@@ -66,6 +66,21 @@ public void checkAddButton() {
                 matches(isDisplayed()));
     }
 
+
+    //Checks adding breakfast to the planner is performed correctly
+    @Test
+    public void BreakfastAddToPlanner() throws InterruptedException {
+        Log.d(TAG, "Testing searching");
+
+        onView(withText("Meal Planner")).perform(click());
+        onView(withId(0)).perform(click());
+
+        Thread.sleep(THREAD_SLEEP_TIME/4);
+
+        onView(withText("Soup")).perform(click());
+        onView(withText("Add")).perform(click());
```

Comment:
This test may fail if the 'soup' recipe is not visible.
Also dosen't guarentee that you are clicking on the 'lunch', 'dinner' or 'breakfast' segment.